### PR TITLE
Updated the magnet example

### PR DESCRIPTION
**Pull request by ddabrahim**
Changed the position of the center and origin point of the big magnet to improve touch experience

### CODE REVIEW

File: sh.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF88ED0ECE7
```
Comment (4ian) [inline sh.exe.stackdump]: You may want to remove this file: `git rm sh.exe.stackdump`, then commit :)

**Comment by 4ian** [comment]
Thanks for the PR :) There is an extra file, check if you can remove it. Otherwise I'll fetch your branch and remove it myself.
Note also that the web-app are separated, and can be updated by launching the script called `update-fixtures-from-resources-examples.js` in folder `newIDE/app/scripts`.
You can launch it with Node.js: 

```
cd newIDE/app/scripts
node update-fixtures-from-resources-examples.js
```

(You must have Node.js installed, which is maybe already the case if you tried to launch the development version of GDevelop).
If you don't manage to do it let me know the error and I'll launch it myself too :)

File: sh.exe.stackdump [hunk 2]
```diff
@@ -1,21 +0,0 @@
-Exception: STATUS_STACK_OVERFLOW at rip=7FF88ED0ECE7
```
Comment (4ian) [inline sh.exe.stackdump]: 👍

**Comment by ddabrahim** [comment]
> the web-app examples are separated, and can be updated by launching the script called update-fixtures-from-resources-examples.js

Tried to launch it but it throw me an error
Error: Cannot find module '../public/libGD.js'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\...\GD\newIDE\app\scripts\update-fixtures-from-resources-examples.js:6:12)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)

**Comment by ddabrahim** [comment]
Did work. Thanks :)

**Comment by 4ian** [comment]
Ahah this time it's because of me, I'm working with a slightly newer version of libGD.js (the core C++ classes of GDevelop compiled to Javascript) where I added support for entering the game version number in the properties window (useful for exports like Cordova and the Windows/macOS/Linux export coming!). As you don't have this version, all the examples were updated to remove this version number.

So no worries I'll take care of this. :)

**Comment by ddabrahim** [comment]
I see.
I was hoping by making a pull request on GitHub I can save you some time :D
So should I reverse the last commit or it doesn't matter,.. Let me know if it does.

**Comment by 4ian** [comment]
No don't change anything, that's great and will already save me a lot of time :) (thanks!)
Will merge this and then the version will be added back on the next examples update.

**Comment by 4ian** [comment]
Merged! Thanks for taking the time to do the PR 👍

**Pull request merged**